materials/spatial-data-R.md

```diff
@@ -140,13 +147,13 @@ extract(chm_harv, plots_harv_utm, buffer = 10, fun = mean)
 
 ```
 plot_latlong_data <- read.csv("data/NEON-airborne/plot_locations/HARV_PlotLocations.csv")
-plot_latlong_data
 crs_longlat <- crs("+proj=longlat +datum=WGS84 +ellps=WGS84 +towgs84=0,0,0")
-plot_latlong_data_spat <- SpatialPointsDataFrame(plot_latlong_data[c('long', 'lat')],
-                                                    plot_latlong_data,
-                                                    proj4string = crs_longlat)
+plot_latlong_data_spat <- SpatialPointsDataFrame(plot_latlong_data[c('long', 'lat')], plot_latlong_data, proj4string = crs_longlat)
```

Comment:
Keeping the arguments on separate lines will help this display more clearly on the website. It's also good practice since this line is >80 characters.